initrd/etc/functions

```diff
@@ -299,7 +275,7 @@ increment_tpm_counter() {
 	TRACE "Under /etc/functions:increment_tpm_counter"
 	tpmr counter_increment -ix "$1" -pwdc '' |
 		tee /tmp/counter-$1 ||
-		die "Counter increment failed"
+		die "TPM counter increment failed for rollback prevention. Please reflash firmware and choose TPM Reset reset option"
```

Comment:
@JonathonHall-Purism well, increment_counter is called from a limited number of places, being in oem-factory-reset and kexec-sign-config which will fail at that point if counter is invalid. That counter being invalid means that the user should reset TPM, but the cause of that counter being invalid is unknown. The only suggestion I can come up with is to flash a firmware image and reset TPM.

@JonathonHall-Purism what else can the user do when he lands there?

Reply:
Resetting the TPM recreates the counter (wiped by resetting TPM, then created by check_tpm_counter), so that should get back to a working state, regardless of the source of the problem.

Even if it somehow didn't get back to a working state, I don't see how flashing the firmware image would help, as it'll still try to do the same thing :shrug:

Reply:
@JonathonHall-Purism : well, reflashing is the only way to make sure but agreed with you that insisting on reflashing a clean firmware would still require TPM Reset.


so I understand that what is desired here is 
`die "TPM counter increment failed for rollback prevention. Please reflash firmware and choose TPM Reset reset option"` -> `die "TPM counter increment failed for rollback prevention. TPM Reset is required to fix this"` ?

Reply:
Right idea, maybe `die "TPM counter increment failed for rollback prevention. Please reset the TPM"` just to be a bit more direct?

Reply:
@JonathonHall-Purism fixed under 19c5d16e403